docs/bridge/bridging/2_permissions.mdx

```diff
@@ -77,3 +77,31 @@ Currently, there are 3 main ways for connector permissions management:
 - Management through the Calimero Console UI
 - Management through the Calimero Bridge SDK
 - Directly through the CLI
+
+### Security and Motivation
+
+There are multiple reasons why bridge permissions need to be set properly:
+1. Avoid malicious entities from bridging FTs/NFTs into private shard
+2. Avoid leaking private information through Cross Shard Calls
+3. Avoid paying connectors gas fees on the destination chain for malicious actor's calls
+4. And potentially more...
```

Comment:
```suggestion
```
I would either rephrase it, or remove it, as it sounds we are hiding some other use cases